docs/components/CardLinks/index.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import NavCard from './CardLink';
```

Comment:
```suggestion
import CardLink from './CardLink';
```

Reply:
I must have renamed the component at some point and forgot to update, how weird that it didn't throw an error 🤯